beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -306,6 +306,9 @@ func bestFinalized() ([]byte, uint64, []peer.ID) {
 	rootToEpoch := make(map[[32]byte]uint64)
 	for _, k := range peerstatus.Keys() {
 		s := peerstatus.Get(k)
```

Comment:
nitpick... `s/peerstatus/peerStatus`

we can fix that later

Reply:
That's the name of the package

Reply:
Ahhh. my mistake